### PR TITLE
Add ISV partners page

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Built ISV partners page as per [copy doc](https://docs.google.com/document/d/1qWWpFbwZykjAIt-Kf6iFCXPCnmmuKBbU5mzvILnO-q0/edit#heading=h.dzb0bs0676v)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/partners/isv-partners
- Check copy against doc
- Test 'become a partner' button and footer links

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4994

### CODE REVIEW

**Review by sowasred2012 — APPROVED** [review]
LGTM - would be worth asking Mark L if it should be added to the secondary nav too (if you haven't already!)

**Comment by mtruj013** [comment]
> LGTM - would be worth asking Mark L if it should be added to the secondary nav too (if you haven't already!)

Thanks @sowasred2012! Just waiting for his reply before I merge :)

**Pull request merged**
